src/cryptoadvance/specter/wallet.py

```diff
@@ -1866,25 +1872,37 @@ def fill_psbt(
         b64psbt,
         non_witness: bool = True,
         xpubs: bool = True,
-        taproot_derivations: bool = False,
+        taproot_derivations: bool = True,
```

Comment:
Does this change of the default make sense? 
It would only add the taproot PSBT fields if it is a taproot wallet. 
But, if a user adds taproot keys by accident / uses them very rarely he would carry those fields with him in the QR codes all the time even if his signing device (devices can also be switched) can't even process the taproot fields.